### ISSUE TITLE
Generate full link for sections

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etlocal#474]
In some cases it is useful to be able to direct clients to specific parts of the Dataset Manager. For example to refer to the Electricity own use in Denmark, which is located in the Energy production > Energy demand section.

<img width="950" alt="Screenshot 2023-05-17 at 14 47 14" src="https://github.com/quintel/etlocal/assets/67338510/32cf8fa4-d0e7-4be5-8998-e34141696548">

When visiting this section, the link remains unchanged: https://data.energytransitionmodel.com/datasets/DK

An idea would be to generate the full link for sections, like we do in the ETM frontend. So something like: https://data.energytransitionmodel.com/datasets/DK/energy-production/energy-demand

@noracato are there any practical considerations not to implement this?
@Charlottevm what do you think of this idea?

**Comment by Charlottevm** [comment]
Yes good idea! I am definitely in favor of referring people to the datasetmanager as much as possible, lets make use of all the information that is there. 

Would it be possible to easily drop a link in ETModel which then 'changes' based on the dataset you open? So if we insert a link in ETModel referring to the electricity own use, it shows this if you open a scenario for Denmark but shows Amsterdam if you were to open a scenario for Amsterdam? @noracato

**Comment by noracato** [comment]
Very possible and not too hard. The url's in ETLocal would take less than a day.

> Would it be possible to easily drop a link in ETModel which then 'changes' based on the dataset you open? 

We already do this in 'sources', so yes, that would be very easy 🙂

**Comment by mabijkerk** [comment]
@noracato is this something @louispt1 would be able to pick up at some point?
